### PR TITLE
Tasks 1 10

### PR DESCRIPTION
**Pull request by YarynaD**

### CODE REVIEW

File: tasks/3_is-boolean/is-boolean.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* eslint "require-jsdoc": 0 */
+
+export function isBoolean(variable) {
+  if (typeof(variable)=== 'boolean') {
+    return true;
+  } else {
+    return false;
+  }
+}
```
Comment (IgorStetsiuk) [inline tasks/3_is-boolean/is-boolean.js]: You can simplify it `return typeof variable === 'boolean'` : ) 
Cause this checking` typeof variable === 'boolean' ` gives us true or false already.

**Comment by IgorStetsiuk** [comment]
You don't have to make changes but keep in mind that it can be useful for future PR's

**Comment by tarasbobak** [comment]
Great comments @IgorStetsiuk 
@YarynaD  Please pay attention to them in future work

**Comment by IgorStetsiuk** [comment]
@tarasbobak Thanks! Going to merge!

**Pull request merged**